tool.ts

```diff
@@ -194,10 +193,13 @@ export function findLocalToolVersions(toolName: string, arch?: string) {
  *
  * @param url       url of tool to download
  * @param fileName  optional fileName.  Should typically not use (will be a guid for reliability). Can pass fileName with an absolute path.
+ * @param handlers  optional handlers array.  Handlers to pass to the HttpClient for the tool download.
```

Comment:
nit: Make it clear handlers are for auth, e.g.

`Auth handlers to pass to the HttpClient for the tool download`